flink-python/pyflink/table/tests/test_pandas_conversion.py

```diff
@@ -132,6 +132,12 @@ def test_from_pandas(self):
                             "1970-01-01 00:00:00.123,[hello, 中文],1,hello,"
                             "1970-01-01 00:00:00.123,[1, 2]"])
 
+    def test_from_pandas_with_float_index(self):
```

Comment:
Is it possible integrate this test case into an existing test case? Then we could avoid introduce another IT case.

Reply:
Yes. Make sense